tests/api/test_soc_descriptor_gs.cpp

```diff
@@ -0,0 +1,149 @@
+#include "gtest/gtest.h"
```

Comment:
Would also love to see a test (which would be arch agnostic, it would run differently on each chip but pass) in api_chip which just creates a driver, and fetches soc_desc and then does something with some coords and calls some driver API (maybe simple write_device).
But you probably need the other PR done which removes requirement for soc_desc in constructor, right? Maybe this test would then belong in that PR.

Reply:
Completely makes sense, thought the same thing. I will write it when I complete removal of soc descriptor

Reply:
Follow up https://github.com/tenstorrent/tt-umd/issues/212